shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -121,7 +121,223 @@ unionClause
     ;
 
 queryBlock
-    : SELECT duplicateSpecification? projections fromClause? whereClause? groupByClause? havingClause?
+    : withClause? SELECT duplicateSpecification? projections fromClause? whereClause? groupByClause? havingClause?
```

Comment:
The rule `duplicateSpecification` is not the same as definition in oracle rule.
![image](https://user-images.githubusercontent.com/22066046/120567102-80577400-c443-11eb-906a-b4ce1361a8dd.png)

Reply:
The rule `projections` is not the same as definition in oracle rule.
![image](https://user-images.githubusercontent.com/22066046/120567289-f6f47180-c443-11eb-8a00-93d371e29fa2.png)

Reply:
The rule `fromClause` is not the same as definition in oracle rule.
![image](https://user-images.githubusercontent.com/22066046/120567572-b9441880-c444-11eb-9c0b-77507c3b08d9.png)

Reply:
The rules: `whereClause`, `groupByClause`, `havingClause`. They need to be checked whether it is the same as the definition of the oracle rule.

Reply:
Hi @wgy8283335, yes the above-mentioned rules aren't aligned with Oracle documentation rules. Since `withClause` contains a lot of changes, I only put `withClause` definition in this PR. I thought to proofread each rule in the `queryBlock` with seperate PRs. What do you think?

Reply:
I agree with you.